k8s/src/main/scala/io/buoyant/k8s/EndpointsNamer.scala

```diff
@@ -224,9 +224,19 @@ abstract class EndpointsNamer(
     // `Activity[Option[Var[Set[Address]]]]`, where the outer `Activity` will
     // update if the `Option` changes, and the inner `Var` will update on
     // changes to the value of the set of `Address`es.
-    stabilize(unstable)
+    val act = stabilize(unstable)
       // convert the contents of the stable activity to a `NameTree`.
       .map { mkNameTree(id, residual) }
+    act.states.respond { state =>
```

Comment:
This will create an observation on the activity that will last forever and hold the activity open.  I don't think we want that.

Reply:
ah, yeah, this was added for debugging purposes and should have been removed, my bad!